### ISSUE TITLE
Video Doesnt Play Only the Example Ones play properply

### ISSUE THREAD

**Issue by nitishpatel** [ja2375/FluTube#10]
![flutter_02](https://user-images.githubusercontent.com/39486276/58380601-a0d8d200-7fd0-11e9-881b-2b9cf43eb829.png)

**Comment by ja2375** [comment]
Sorry to hear :(
Which video are you trying to play?
Does it throw any exception?

**Comment by nitishpatel** [comment]
yes!
Video url:[https://www.youtube.com/watch?v=NHr6lTKYCNc&t=20s](url)

**Comment by ja2375** [comment]
Can you paste the exception stack trace here?

**Comment by nitishpatel** [comment]
```
An Observatory debugger and profiler on Redmi Note 5 is available at: http://127.0.0.1:61559/i9qIX6Tkz6c=/
For a more detailed help message, press "h". To detach, press "d"; to quit, press "q".

I/ExoPlayerImpl(25890): Init cc2f390 [ExoPlayerLib/2.9.6] [vince, Redmi Note 5, Xiaomi, 27]
D/NetworkSecurityConfig(25890): No Network Security Config specified, using platform default
W/zygote64(25890): miui_dex2oat: oat file of /system/framework/tcmclient.jar is not exists
W/zygote64(25890): miui_dex2oat: /system/framework/tcmclient.jar: Fall back to running out of the original dex file.
I/DpmTcmClient(25890): RegisterTcmMonitor from: com.android.okhttp.TcmIdleTimerMonitor
I/DpmTcmClient(25890): RegisterTcmMonitor from: com.android.okhttp.TcmIdleTimerMonitor
E/ExoPlayerImplInternal(25890): Source error.
E/ExoPlayerImplInternal(25890): com.google.android.exoplayer2.upstream.HttpDataSource$InvalidResponseCodeException: Response code: 403
E/ExoPlayerImplInternal(25890):         at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.open(DefaultHttpDataSource.java:300)
E/ExoPlayerImplInternal(25890):         at com.google.android.exoplayer2.upstream.StatsDataSource.open(StatsDataSource.java:83)
E/ExoPlayerImplInternal(25890):         at com.google.android.exoplayer2.source.ExtractorMediaPeriod$ExtractingLoadable.load(ExtractorMediaPeriod.java:885)
E/ExoPlayerImplInternal(25890):         at com.google.android.exoplayer2.upstream.Loader$LoadTask.run(Loader.java:381)
E/ExoPlayerImplInternal(25890):         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
E/ExoPlayerImplInternal(25890):         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
E/ExoPlayerImplInternal(25890):         at java.lang.Thread.run(Thread.java:764)
```

**Comment by ja2375** [comment]
Please update to 0.8.0 and see if this gets fixed for you. If not, please reopen the issue.
